src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,49 @@
+@use "../utils/variables" as *;
+
+.stars {
+  display: flex;
+  margin-top: 16px;
+  justify-content: center;
+  align-items: center;
+  &__star {
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: $font-weight-normal;
```

Comment:
The variable `$font-weight-normal` is used here, but it is not defined or imported from the `variables` file. Please ensure that this variable is defined in the `../utils/variables` file or replace it with a valid font-weight value like `400`.